src/FileException.h

```diff
@@ -32,7 +32,7 @@ class AUDACITY_DLL_API FileException /* not final */
       const TranslatableString &caption = XO("File Error"), //!< Shown in message box frame, not the main message
       const wxFileName &renameTarget_ = {} //!< A second file name, only for renaming failure
    )
-   : MessageBoxException{ caption }
+   : MessageBoxException{ ExceptionType::Internal, caption }
```

Comment:
But now I wonder if we always want this to be Internal too.  What if you are just trying to write to a folder that is write-protected or a drive that is out of space?

You decide.  Write a comment if you don't change it.

Reply:
>What if you are just trying to write to a folder that is write-protected

We had peculiar SQLite errors in this case in 3.0.

Let's leave it for now and see, how it turns out.